README.md

```diff
@@ -1,2 +1,397 @@
+
+---
+
+# BrowserStack
+
+App & Browser Testing Made Easy!! Give your users a seamless experience by testing on 2000+ real devices and browsers. Don't compromise with emulators and simulators.
+
+## Prerequisites
+
+- Create an account on BrowserStack if not already
+- Export the environment variables for the Username and Access Key of your BrowserStack account.
+
+    - For \*nix based and Mac machines:
+
+  ```sh
+  export BROWSERSTACK_USERNAME=<browserstack-username> &&
+  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+    - For Windows:
+
+  ```shell
+  set BROWSERSTACK_USERNAME=<browserstack-username>
+  set BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+- Clone the browserstack-canonical-app repository using `git clone https://github.com/browserstack/browserstack-demo-app`
+- Update the [caps.json](resources/conf/caps/caps.json) file according to your device/browser needs. You can find the capabilities needed from here: [browserstack-capability-generator](https://browserstack.com/automate/capabilities)
+
+## Run Your Tests
+
+In this section we will run tests on BrowserStack infrastructure
+
+### Run Single Test on Browserstack
+
+In this section we will run a single the test on chrome on Browserstack. To change Capabilities for this configuration refer to `single` object in `caps.json` file.
+
+- How to run the test?
+
+  To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn install -P bstack-single -Dtest-name="test name"
+  ```
+
+- Output
+
+  For output it will run test on browserstack. Check attached screenshots below.
+
+    - Console output
+
+  ```sh
+  ~paste console output!~
+  ```
+
+    - Dashboard Screenshot
+      <img src="dashboard.png" height="289.6" width="568.4" >
+
+  ~img res /5 should be height and width~
+
+### Run Tests in parallel on single browser on Browserstack
+
+In this section we will run the tests in parallel on single browser on Browserstack. To change Capabilities for this configuration refer to `single` object in `caps.json` file.
+
+- How to run the test?
+
+  To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn install -P bstack-parallel
+  ```
+
+- Output
+
+  For output it will run tests on browserstack. Check attached screenshots below.
+
+    - Console output
+
+  ```sh
+  ~paste console output!~
+  ```
+
+    - Dashboard Screenshot
+      <img src="dashboard.png" height="289.6" width="568.4" >
+
+  ~img res /5 should be height and width~
+
+### Run Tests in parallel on multiple browser on Browserstack
+
+In this section we will run the tests in parallel on multiple browser on Browserstack. To change Capabilities for this configuration refer to `parallel` object in `caps.json` file.
+
+- How to run the test?
+
+  To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn compile exec:java -P bstack-parallel-browsers
```

Comment:
this is all what came on the output. no tests ran:

➜  browserstack-examples-cucumber-junit5 git:(develop) ✗ mvn compile exec:java -P bstack-parallel-browsers
[INFO] Scanning for projects...
[INFO]
[INFO] -------< com.browserstack:browserstack-examples-cucumber-junit5 >-------
[INFO] Building browserstack-examples-cucumber-junit5 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ browserstack-examples-cucumber-junit5 ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /Users/samiran/Documents/se/repos/test-university/browserstack-examples-cucumber-junit5/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ browserstack-examples-cucumber-junit5 ---
[INFO] No sources to compile
[INFO]
[INFO] --- exec-maven-plugin:3.0.0:java (default-cli) @ browserstack-examples-cucumber-junit5 ---
{"env_caps":[{"os":"Windows","os_version":"10","browser":"Internet Explorer","browser_version":"11.0"},{"os":"Windows","os_version":"10","browser":"Chrome","browser_version":"latest-1"},{"os":"OS X","os_version":"Big Sur","browser":"Chrome","browser_version":"latest"},{"os":"OS X","os_version":"Catalina","browser":"Safari","browser_version":"13.0"},{"os":"Android","realMobile":true,"os_version":"11.0","device":"Samsung Galaxy S20"},{"os":"iPhone","realMobile":true,"os_version":"13.0","device":"iPhone XS"}],"common_caps":{"build":"Automate Demo","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","browserstack.debug":"true","browserstack.console":"verbose"}}
{"capabilities":{"os":"Windows","build":"Automate Demo","os_version":"10","browser":"Internet Explorer","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","browser_version":"11.0","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
{"capabilities":{"os":"Windows","build":"Automate Demo","os_version":"10","browser":"Chrome","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","browser_version":"latest-1","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
{"capabilities":{"os":"OS X","build":"Automate Demo","os_version":"Big Sur","browser":"Chrome","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","browser_version":"latest","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
{"capabilities":{"os":"OS X","build":"Automate Demo","os_version":"Catalina","browser":"Safari","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","browser_version":"13.0","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
{"capabilities":{"os":"Android","build":"Automate Demo","realMobile":"true","os_version":"11.0","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","device":"Samsung Galaxy S20","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
{"capabilities":{"os":"iPhone","build":"Automate Demo","realMobile":"true","os_version":"13.0","name":"Parallel Test: ","browserstack.networkLogs":"true","project":"BrowserStack Demo","device":"iPhone XS","browserstack.debug":"true","browserstack.console":"verbose"},"application_endpoint":"https:\/\/bstackdemo.com","user":"BROWSERSTACK_USERNAME","key":"BROWSERSTACK_ACCESS_KEY"}
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.716 s
[INFO] Finished at: 2021-03-16T19:59:10+05:30
[INFO] ------------------------------------------------------------------------

Reply:
fixed